package-lock.json

```diff
@@ -1,7 +1,14824 @@
 {
```

Comment:
Your PR should not contain any changed to the package-lock.json file.

Reply:
I think this change was made by the GitHub actions bot when I commented `/format` in the PR as I could not format it locally.
Is there a way I can revert changes in only one file?

Reply:
@SleeplessByte Do you know why the formatting workflow suddenly started to change the package.json file? Some new npm version and the file is not up-to-date maybe?

@JaPatGitHub You can revert the change like this https://fettblog.eu/snippets/git/reverting-a-single-file/ but you need to find the right commit for this. So it might be faster just to copy and paste from main and then commit that change.

Reply:
![image](https://user-images.githubusercontent.com/1964376/147928613-a733e9f8-782d-432d-81bd-bd431a6f71be.png)

This is why, so yes.

Reply:
I proposed a fix here: https://github.com/exercism/javascript/pull/1607